### ISSUE TITLE
[MAS2.4.7] [Keyboard Navigation -Live Chat] Visual focus indicator is not visible on File upload button present in the Live chat screen 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1753]
**Actual Result:**
Visual focus indicator is not visible on File upload button present in the Live chat screen

**Expected Result:**
 Visual focus indicator should be visible on File upload button 

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3.Navigate to the open bot and select it  
4. Dialog gest opened and connect the open bot  
5. Live chat screen gets opened 
6.Navigate to the File upload button using Tab key 
7. Observe the issue. 

**User Impact:**
users who rely on Keyboard navigation will be affected the keyboard focus indicator is not visible on control so that keyboard dependant users will be affected to locate where exact position of keyboard focus.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[Sharepoint Link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})
[MAS2.4.7_Live chat.zip](https://github.com/microsoft/BotFramework-Emulator/files/3519958/MAS2.4.7_Live.chat.zip)
![MAS2 4 7_Live Chat](https://user-images.githubusercontent.com/53518994/63338917-65486c80-c361-11e9-81a6-631314765b29.png)

**Comment by corinagum** [comment]
I think this might be a good one to have in Web Chat, and let it trickle down to Emulator. 
We can provide the ability for the user to choose the focus appearance in `defaultStyleOptions`

**Comment by dollykumari5** [comment]
@corinagum This bug is logged for visual focus indicator on the file upload button just next to the text box. That button should have focus indicator such as a rectangle box around it when focus comes on it.
And for the webchat text box we are good if we are having a vertical blinking line on it as focus similar to that of any text field.

**Comment by Kumar2608** [comment]
Verified in latest bot emulator  Version 4.5.2-78641 and issue got fixed Hence closing this bug 
Now VFI is clearly visible on File upload button.
